ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -0,0 +1,139 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ebuttdt="urn:ebu:tt:datatypes"
+	targetNamespace="urn:ebu:tt:datatypes">
+	<xs:simpleType name="cellResolutionType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="[0]*[1-9][0-9]*\s[0]*[1-9][0-9]*"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="frameRateMultiplierType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="[0]*[1-9][0-9]*\s[0]*[1-9][0-9]*"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="extentType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="fontFamilyType">
+		<xs:restriction base="xs:string"/>
+	</xs:simpleType>
+	<xs:simpleType name="fontSizeType">
+		<xs:restriction base="xs:token">
+			<xs:pattern
+				value="(\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+			/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="lineHeightType">
+		<xs:union memberTypes="ebuttdt:lengthType">
+			<xs:simpleType>
+				<xs:restriction base="xs:token">
+					<xs:enumeration value="normal"/>
+				</xs:restriction>
+			</xs:simpleType>
+		</xs:union>
+	</xs:simpleType>
+	<xs:simpleType name="colorType">
+		<xs:union
+			memberTypes="ebuttdt:rgbHexColorType ebuttdt:rgbaHexColorType ebuttdt:rgbColorType ebuttdt:rgbaColorType ebuttdt:namedColorType"
+		/>
+	</xs:simpleType>
+	<xs:simpleType name="originType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="paddingType">
+		<xs:restriction base="xs:token">
+			<xs:pattern
+				value="(\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+			/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="timingType">
+		<xs:union
+			memberTypes="ebuttdt:smpteTimingType ebuttdt:mediaTimingType ebuttdt:clockTimingType"/>
+	</xs:simpleType>
+	<xs:simpleType name="durationTimingType">
+		<xs:union 
+			memberTypes="ebuttdt:clockTimingType ebuttdt:mediaTimingType"/>
+	</xs:simpleType>
+	<xs:simpleType name="smpteTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-2][0-9]:[0-5][0-9]:[0-5][0-9]:[0-9][0-9]"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="clockTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-9][0-9]:[0-5][0-9]:[0-5][0-9]:[0-9][0-9]"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="mediaTimingType">
+		<xs:union memberTypes="ebuttdt:timecountTimingType ebuttdt:fullClockTimingType"/>
+	</xs:simpleType>
+	<xs:simpleType name="timecountTimingType">
```

Comment:
This is a bit misleading, since it's a really specific kind of timecount timing type that only has applicability on the `authoringDelay` attribute. What do you think about renaming it to authoringDelayType?